status/profile_plots.py

```diff
@@ -25,26 +25,26 @@ def wrapper(*args, **kwargs):
         t0 = time.time()
```

Comment:
This `time_function` appears to be defined, but not used by any code.  It seems more in line with debugging functionality, but we should also be able to get execution time from `cProfile`.  I'd consider removing this function entirely.